### ISSUE TITLE
Cloudflare SSL Certificate

### ISSUE THREAD

**Issue by incoming-th** [dunglas/frankenphp#602]
### Describe you feature request

Hi

More a question than a feature as this may be available already.

**Is your feature request related to a problem? Please describe.**
Let's Encrypt is nice but we prefer to use SSL Certificate from Cloudflare which are free and valid for 15 years.

**Describe the solution you'd like**
It would be handy to have an option that automatically disables Let's Encrypt by providing the path to the private and public key files where we saved the Cloudflare Certificate.

Apologies if this is already possible as may have missed it in the doc.

Thanks

**Comment by dunglas** [comment]
This is supported by Caddy: https://caddyserver.com/docs/caddyfile/directives/tls#load

**Comment by incoming-th** [comment]
@dunglas Thanks for this, but I may need an ELI5 here.

Should we put the parameter in the frankenphp line? What should be the files named?

```./frankenphp php-server --worker /path/to/your/worker/script.php --load /path/to/ca/folder```

Thanks

**Comment by dunglas** [comment]
No, you need to use a custom Caddyfile to achieve this.

**Comment by DatPenguin** [comment]
Hi, sorry to comment on a closed issue but I've done precisely that (creating a custom Caddyfile and mounting it in `/etc/caddy/Caddyfile`) and it does not seem to work : there is still a Let's Encrypt challenge going on, which is obviously going to fail since I am inside a private domain.

The Caddyfile contains the load :
```
tls {
        load /opt/certs
 }
```
`/opt/certs` contains a `.pem` file with my cert and key.

Is there something more to do to ensure this cert is taken into account and Let's Encrypt not run ?

Thanks
